enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/AbstractPropertyEnforcerRule.java

```diff
@@ -32,16 +32,16 @@ public abstract class AbstractPropertyEnforcerRule extends AbstractNonCacheableE
     /**
      * Match the property value to a given regular expression. Defaults to <code>null</code> (any value is ok).
      *
-     * @see {@link #setRegex(String)}
-     * @see {@link #getRegex()}
+     * @see #setRegex(String)
+     * @see #getRegex()
      */
     private String regex = null;
 
     /**
      * Specify a warning message if the regular expression is not matched.
      *
-     * @see {@link #setRegexMessage(String)}
-     * @see {@link #getRegexMessage()}
+     * @see #setRegexMessage(String)
+     * @see #getRegexMessage()
```

Comment:
Why such change ... it is connected somehow?
😄 
I would like to do such changes in separate PR if possible.

Reply:
My linter complained about it, but can be done in a separate PR indeed.